addons/web/static/src/core/notifications/notification.scss

```diff
@@ -1,47 +1,13 @@
-$notification-width: 320px;
-$notification-text-color: #212529;
-$notification-title-text-color: #6c757d;
-
 // Colors of the notification are not managed by this file.
 // It is handled by the other classes defining a background. (like danger, warning, etc.)
 // The notification just add overlays.
 
 .o_notification_manager {
-  @include o-position-absolute(3.5rem, 0);
-  position: fixed;
+  @include o-position-absolute($o-navbar-height * 1.15, map-get($spacers, 2), $left: calc(100vw - #{$toast-max-width}));
```

Comment:
@xlu-odoo Not sure to understand why you choose left instead of right.

Reply:
@adr-odoo 
I think it is how it is supposed to work.. 
When using the calc and a max-width for the notifications, it pushes everything starting the left. 
(It's one line less than using one for a specific width, and one for a `right: **` )

Reply:
Well actually not really "one line less" as the mixin generates 5 rules every time... :wink: